wirerope/callable.py

```
@@ -1,9 +1,8 @@
 from __future__ import absolute_import
```

Ruff (0.15.15)

[warning] Unnecessary `__future__` import `absolute_import` for target Python version

Remove unnecessary `__future__` import

(UP010)